private-api/app/src/VotingIrregularities.Domain/Migrations/20190420140225_FormEntityAdjustments.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```

Comment:
@ExtremeSwat these migration files should go in src/VotingIrregularities.Domain, I think you missed them in the merge to the 2.2 upgrade